docs/docs/guides/smart_contracts/deploying_and_interacting_with_smart_contracts.md

```diff
@@ -353,6 +353,56 @@ Transaction Hash: 0x9825e2a2115896728d0c9c04c2deaf08dfe1f1ff634c4b0e6eeb2f504372
 my number updated value: 2
 ```
 
+## Troubleshooting and errors
+
+If you are running into errors when executing contract methods such as `myContract.methods.call` or `myContract.deploy.estimateGas()` you might be see a contract execution revert error such as: `value transfer did not complete from a contract execution reverted`.
+
+This could be due to the node you are connected to and is expecting the `data` property to be populated in your contract instead of `input`. Web3 version >4.03 will always populate `input` when sending transactions.
```

Comment:
I think the version `4.03` has a typo.
And can we give an example for the provider that has this issue?

Reply:
updated, thank you